clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -514,6 +521,23 @@ private void handleColumnCommand() {
     }
   }
 
+  private void handleHelpCommand() {
+    String helpFile = entity.toLowerCase() + "_help.txt";
```

Comment:
will each entity type has a help.txt file? here only see the metalake_help.txt

Reply:
That is covered by other issues - please see the parent issue.

Reply:
Also, being good first issues, others will pick them up, and/or I'll get to them after this PR has been merged and the relevant code PR has been merged.

Reply:
the owner code has been merged